File: sdkwork-ai-sdk-java/src/main/java/com/sdkwork/ai/model/ChunkConfig.java
package com.sdkwork.ai.model;

public class ChunkConfig {
    private String separator;
    private Integer chunkSize;
    private Integer chunkOverlap;

    public String getSeparator() {
        return this.separator;
    }
    
    public void setSeparator(String separator) {
        this.separator = separator;
    }

    public Integer getChunkSize() {
        return this.chunkSize;
    }
    
    public void setChunkSize(Integer chunkSize) {
        this.chunkSize = chunkSize;
    }

    public Integer getChunkOverlap() {
        return this.chunkOverlap;
    }
    
    public void setChunkOverlap(Integer chunkOverlap) {
        this.chunkOverlap = chunkOverlap;
    }
}

File: sdkwork-ai-sdk-csharp/Models/ChunkConfig.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class ChunkConfig
    {
        public string? Separator { get; set; }
        public int? ChunkSize { get; set; }
        public int? ChunkOverlap { get; set; }
    }
}
